plugins/woocommerce-admin/client/marketing/data-multichannel/actions.ts

```diff
@@ -38,9 +43,41 @@ export const receiveRecommendedChannelsError = ( error: ApiFetchError ) => {
 	};
 };
 
+export const receiveCampaignsSuccess = ( success: {
+	payload: Array< Campaign >;
+	error: boolean;
+	meta: {
+		page: number;
+		perPage: number;
+		total?: number;
+	};
+} ) => {
+	return {
+		type: TYPES.RECEIVE_CAMPAIGNS_SUCCESS,
+		...success,
+	};
+};
+
+export const receiveCampaignsError = ( error: {
```

Comment:
Missed JSDOCS

Reply:
Done in [`4b87f0a` (#36735)](https://github.com/woocommerce/woocommerce/pull/36735/commits/4b87f0a4ed13d00bd8054687bdab823ba10a6a03).